Kickstarter-iOS/Views/Controllers/SettingsPrivacyViewController.swift

```diff
@@ -59,11 +59,18 @@ internal final class SettingsPrivacyViewController: UITableViewController {
         self?.present(UIAlertController.genericError(message), animated: true, completion: nil)
     }
 
-    self.viewModel.outputs.refreshFollowingSection
+    self.viewModel.outputs.resetFollowingSection
       .observeForUI()
       .observeValues { [weak self] _ in
-        let followingSection = IndexSet(integer: Section.following.rawValue)
-        self?.tableView.reloadSections(followingSection, with: .none)
+        let indexPath = IndexPath(row: 0, section: Section.following.rawValue)
+        let cell = self?.tableView.cellForRow(at: indexPath) as? SettingsFollowCell
+        cell?.toggleOn()
+    }
+
+    self.viewModel.outputs.focusScreenReaderOnFollowingCell
```

Comment:
Explicit signal instead of a side effect previously when we had this in `resetFollowingSection`